src/components/details/PokemonDetails.tsx

```diff
@@ -1,15 +1,53 @@
 import { StatusBar } from 'expo-status-bar';
-import { SafeAreaView, StyleSheet, Text, View } from 'react-native';
+import { ActivityIndicator, SafeAreaView, StyleSheet, Text } from 'react-native';
+import React, { useState, useEffect } from 'react';
+import { View } from 'react-native';
 
 
-export function PokemonDetails({ route, navigation }: { route: any, navigation: any }) {
+
+export function PokemonDetails({ route }: { route: any, navigation: any }) {
```

Comment:
Are these type-able?

Reply:
Will look into it; I did not realise this before, but is the typing of navigation valid in the second {}, considering that the props was not destructured in the first {}?

Reply:
Yeah it does seem wrong to me. Weird that it is compiling still though. We can ask someone better than me at typing + react native if you're stuck cause I'm not too sure form looking at this